src/stores/clusterapps/actions.ts

```diff
@@ -0,0 +1,65 @@
+import GiantSwarm, { V4App } from 'giantswarm';
+import { FlashMessage, messageTTL, messageType } from 'lib/flashMessage';
+
+import { createAsynchronousAction } from '../asynchronousAction';
+
+interface IRequest {
```

Comment:
Is it ok to keep these names super short like this? or will that conflict down the line? my typescript mental still isn't complete.

Reply:
As long as they don't contradict with any global interface, then it's fine.

If it was to contradict with another interface, then it would actually extend it.

Reply:
Ah ok, so having a more specific name would be better in this case :+1: